utils/validation/user.ts

```diff
@@ -0,0 +1,108 @@
+import { locale, roles, province } from ".prisma/client";
+import { UserInput, ParentInput, VolunteerInput } from "models/User";
+import validator from "validator";
+
+/**
+ * Valdaites whether a given province has been specified correctly
+ * @param userProvince - the province to validate
+ * @returns - true if the province is valid, false otherwise
+ */
+function validateProvince(userProvince) {
```

Comment:
Does the Validator library not have a province Validator?

Reply:
No, and either way I think maybe it would be good to compare against the values we are using in our database just to ensure that it's consistent